wherescape/connectors/hubspot/python_hubspot_check_missing_columns

```diff
@@ -0,0 +1,130 @@
+"""
+Script that compares the columns from the Hubspot api with the columns in the
+WhereScape load tables. It will load all missing columns in a load table and
+compares it to the ds tables. If new ones have appeared, it add them to the
+WhereScape warning log messages so they turn up in the daily report in Slack.
+"""
+import logging
+from hubspot.crm.properties.exceptions import ForbiddenException
+
+from wherescape import WhereScape
+from wherescape.connectors.hubspot import Hubspot
+
+
+OBJECT_TYPES_TO_CHECK = (
```

Comment:
I would have preferred a solution where this module contains a function that can be called from the code that retrieves the data. Some considerations include:
- you're then able to just use wherescape.columns  
- you cannot forget to add the object to this Tuple
- you're not forcing the open source community to follow our naming standard